compose-files/staff/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+
+networks:
+  placeos_placeos:
```

Comment:
`placeos_placeos`?

Reply:
yep that's the bridge network's name when partner-environment is spun up, docker creates it at <[project_name](https://github.com/place-labs/partner-environment/blob/master/.env#L1)>_<[network name](https://github.com/place-labs/partner-environment/blob/master/docker-compose.yml#L5)>
and in this case both of those are `placeos`

Reply:
That's only if the network has not been explicitly [named](https://docs.docker.com/compose/compose-file/compose-file-v3/#name-1), which [this one has](https://github.com/place-labs/partner-environment/blob/df3924cea6fbf1def1115bec05c97f9919984206/docker-compose.yml#L5).